src/views/CardStack.js

```diff
@@ -65,7 +69,7 @@ type Props = {
 /**
  * The duration of the card animation in milliseconds.
  */
-const ANIMATION_DURATION = 200;
+const ANIMATION_DURATION = 250;
```

Comment:
It's a approximate value from `src/views/TransitionConfigs.js`, I assume the time for swipe a full screen is equal the `goBack` duration, although that value is configurable via `TransitionConfig`, but actually if we swipe back, the distance is less then the full distance, so the duration should be always less then 250ms, and I think it a proper value, unless we can configure it or read from `transitionConfig`
So I'd prefer to update the comment right now